### ISSUE TITLE
Need a better way of highlighting messages, like bold or underline or simple reverse.

### ISSUE THREAD

**Issue by highflyer77** [weechat/weechat#366]
Highlighting of messages on weechat changes the color of the nick.  That is not desired when using nick coloring options.  Changing the color of the nick makes the thread harder to follow on screen.  It would seem much more desireable to just maybe:  Bold the nick  or  Underline the nick  or a simple reverse color of nick (reverse bg and fg colors), or even underline the whole line of text. It would also be nice to highlight the word that triggered the nick, with possibly the same  or different options.

Specifically these options are too limited:
weechat.color.chat_highlight
weechat.color.chat_highlight_bg
weechat.color.chat_nick_offline_highlight
weechat.color.chat_nick_offline_highlight_bg

or being able to change one of these options on a highlight might help too:
weechat.color.chat_nick_prefix
weechat.color.chat_nick_suffix

Other clients can do this by default, if you guys like this please comment on it. 
Just trying to suggest making the screen more user friendly.
Please consider this feature request, thank you.
Thanks --     WeeChat Rocks!

**Comment by sim642** [comment]
There are kind-of options to set text attributes (bold, etc) via colors like https://weechat.org/files/doc/stable/weechat_user.en.html#colors_attributes. But the problem with those is that you have to specify a color as well. Maybe the solution would be to not require the color and just allow setting a color to `*` to make it bold while leaving whatever color it was supposed to be without the highlight.

**Comment by sim642** [comment]
Per my suggestion, it'd essentially mean a "no-op" color code (with attributes supported still) like suggested by #494. In this case setting no color but attribute should keep the random nick color.

**Comment by bendem** [comment]
Would love that. I'm currently using `weechat.color.chat_highlight color /default`, would much prefer `weechat.color.chat_highlight color /`.

**Comment by quite** [comment]
A no-op colour to put on chat_highlight would be useful. I think I could stand any kind of foreground-background combination, as long as nick foreground remains what the nick_color-system has chosen for it. If a highlighted nick changes foreground colour, that just confuses me.

(My terminal colours are already adjust so that mixing and matching fg/bg colours still renders something which is at least somewhat readable -- and that could even be trimmed a bit more)

**Comment by highflyer77** [comment]
If underlining or something is too hard to code in. I'd be happy with **"No Defaults"**, to just bypass changing the color at all.  That way the hotlist would still work, and the nick colors could remain the same.

**Comment by oakkitten** [comment]
yes! yes. also something could be done about offline nicknames (`weechat.look.color_nick_offline`)

**Comment by arza-zara** [comment]
A hack:

- https://github.com/arza-zara/weechat/tree/nohlprefix
- colorize_lines.pl
- `/trigger add nohlprefix modifier "weechat_print" "${tg_tags} =~ notify_message && ${tg_message} =~ yournick" "/\x19F@/\x19F@*/"`

![highlight_color](http://weechat.arza.us/highlight_color.png)
